config/config.go

```diff
@@ -156,6 +156,11 @@ var (
 		RefreshInterval: model.Duration(5 * time.Minute),
 	}
 
+	// DefaultECSSDConfig is the default ECS SD configuration.
+	DefaultECSSDConfig = ECSSDConfig{
+		RefreshInterval: model.Duration(60 * time.Second),
```

Comment:
Arguably there are quite a few supported right now that use the same mechanism, but I would even suggest converting those into their own projects (thinking towards 2.0 maybe).